src/battle_arcade_game_board.c

```diff
@@ -568,8 +568,8 @@ static void BattleArcadeGameBoard_InitObjects(GAME_BOARD_WORK *work) {
         work->unk78[i] = String_New(32, HEAP_ID_GAME_BOARD);
     }
 
-    LoadFontPal0(GF_BG_LYR_MAIN_0, 416, HEAP_ID_GAME_BOARD);
-    LoadFontPal1(GF_BG_LYR_MAIN_0, 384, HEAP_ID_GAME_BOARD);
+    LoadFontPal0(GF_PAL_LOCATION_MAIN_BG, GF_PAL_SLOT_OFFSET_13, HEAP_ID_GAME_BOARD);
```

Comment:
Question- does GF_PAL_SLOT_OFFSET_X mean the offset to the Xth indexed palette? Ie this function is loading in palette data to palette 13?

Reply:
each palette slot is 0x20 in size. offset 13 loads into the 13th palette (zero indexed)

Reply:
(for context, slot 13 loads at 0x1A0)

Reply:
Would something like PAL_SLOT_13_OFFSET be more clear then?

Reply:
GF_PAL_SLOT_13_OFFSET? could possibly work